dll/cpl/sysdm/general.c

```diff
@@ -543,6 +551,31 @@ static VOID GetSystemVersion(HWND hwnd)
     HeapFree(GetProcessHeap(), 0, pwszStr);
 }
 
+static VOID GetSystemUptime(HWND hwnd)
+{
+    HWND hRosUptime;
+    WCHAR szBuf[64], szStr[64];
+    UINT cSeconds;
+
+    hRosUptime = GetDlgItem(hwnd, IDC_ROSUPTIME);
+    if (!hRosUptime)
+    {
+        return;
+    }
+    if (!LoadString(hApplet, IDS_UPTIME_FORMAT, szStr, _countof(szStr)))
+    {
+        StringCchCopyW(szStr, _countof(szStr), L"%u Days, %u Hours, %u Mins, %u Secs");
```

Comment:
Agreed.
If a string cant be loaded there is a serious error, and in that case fail (without crashing) is better than hiding the problem.

Reply:
A general comment about that: Due to the fact we include english translations conditionally (we need the `LANGUAGE_EN_US` to be defined), it may happen that fallback translations (that switch usually to english) could not find the english resources if that `LANGUAGE_EN_US` is not defined (this is the case e.g. when @JoachimHenze compiles ROS in german-only).